meetings/2021-07/july-15.md

```diff
@@ -0,0 +1,734 @@
+
+MM: Write the proposal, you're referring to is get in transits proposal. We Sure, yes. Okay, Good question.
+
+USA: Is it something that you'd like a temperature check on. 
+
+JHD: I mean, a temperature check seems nice, I think specifically if it would, even though Everyone is always allowed at any time to say, I have an objection or I have a constraint, even though I never mentioned it before, it would feel really bad if I put in time on a proposal like this and then came back in six months, a year, two months, whatever, and was surprised with a sudden constraint, it's happened to me once before it is not a fun thing to do and it's very demotivating. So I really just want to make sure that if there's any possibility that any one, could guess that, that might happen that I could get some hints about it now so that I know whether it's worth putting in my time, this is not really a technical or procedural question, I'm sort of asking you a personal question in somebody. Make sure that I'm not going to about to waste my time.
+
+CP: I don't honestly, I don't see any problem with that moving forward, we have plenty of experience on that, and the only thing that I can offer, there is collaboration. So we can put some effort as well to try to push for it, because this is the reality that we are on when you write libraries right? You have to cache all these things. Otherwise you're just going to be trouble and SES of as well, does that? So I think it's sufficient manpower to work toward. These are such as it's a proposal. 
+
+LEO: From my reading as well. I think you re very welcomed at the size of the group to continue a lot of discussion for this and making sure like, we provide feedback and all the work that we need to, to move this forward because I think Mark would be interested. I am assuming Mark who can confirm that right now. 
+
+MM: Yeah. So, absolutely definitely interested in collaborating on this proposal and seeing it move forward. We cannot write a session that stays secure into the future without something like this.
+
+USA: one question from Greg, from the chat is Jordan are you looking for implementer concern?
+
+JHD: Yeah, I mean in particular, since the of the friction that Realms at that has been implemented concerned. So I want to I, particularly for that. Thank you Greg.
+
+AKI: Real quick. I just want to make sure that you take the time to throw. probably wouldn't want to talk. Technical things are not supposed to be on discourse, maybe try to reach out to people who weren't able to make this kind of very specific time zone Restricted meeting time looking for the set that same feedback of find out ahead of time. If there would be any concerns or if or if there would be any Issues that you should be prepared for before you get started. This could make sure to ask outside of this meeting. There's a lot of people who couldn't be here, who may or may not have opinions. I have no idea. 
```

Comment:
IDK how I both messed this one up and missed it on first review. Extremely false.

```suggestion
```